src/main/java/org/jboss/forge/arquillian/command/cube/KubernetesCubeTestSetup.java

```diff
@@ -0,0 +1,52 @@
+package org.jboss.forge.arquillian.command.cube;
+
+
+import io.fabric8.kubernetes.api.model.Service;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.forge.roaster.model.source.JavaClassSource;
+
+
+public class KubernetesCubeTestSetup implements CubeTestSetup {
+
+    @Override
+    public JavaClassSource updateTest(JavaClassSource test) {
+
+        addImports(test);
+        createEnrichments(test);
+        createTestMethod(test);
+
+        return test;
+    }
+
+    private void createTestMethod(JavaClassSource test) {
+        if (!test.hasMethodSignature("public serviceInstanceShouldNotBeNull() : void")) {
+            test.addMethod()
+                .setName("serviceInstanceShouldNotBeNull")
+                .setPublic()
+                .setReturnTypeVoid();
+        }
+
+        test.getMethod("serviceInstanceShouldNotBeNull")
+            .setBody("assertNotNull(service);")
+            .addAnnotation("Test");
+    }
+
+
+    private void addImports(JavaClassSource test) {
+        test.addImport(ArquillianResource.class);
+        test.addImport("org.arquillian.cube.kubernetes.annotations.Named");
```

Comment:
The reason why we rely on strings is that we don't have it on the classpath at this point?